bindings/node/addon/mongocrypt.h

```diff
@@ -18,97 +18,94 @@ extern "C" {
 namespace node_mongocrypt {
 
 struct MongoCryptBinaryDeleter {
-    void operator()(mongocrypt_binary_t* binary) {
+    void operator()(mongocrypt_binary_t *binary) {
```

Comment:
> We rely on the `.clang-format` file in the project root

… which is targeting C code and not C++, leading to changes like this?

Reply:
Neal and I thought that the C format style would also apply to c++.  Are there different conventions?  I can revert and reformat the bindings using the old clang format file.

Reply:
I mean, at the end of the day linting rules are personal preferences and you can do whatever you prefer, it's like single vs double quotes in JS.

That being said, yes, it's been a long time since I've seen left-leaning asterisks for pointers in C or right-leaning ones in C++, and, very personally, yes this does just look a bit _odd_ to me. (Random example if you google this: https://stackoverflow.com/questions/2660633/declaring-pointers-asterisk-on-the-left-or-right-of-the-space-between-the-type even uses "C style" and "C++ style" to refer to the two styles, respectively).